features/flows/classification/engine/api/src/main/java/org/opennms/netmgt/flows/classification/ClassificationRequest.java

```diff
@@ -69,11 +71,11 @@ public Protocol getProtocol() {
         return protocol;
     }
 
-    public void setDstPort(int dstPort) {
+    public void setDstPort(final Integer dstPort) {
```

Comment:
If we change it from `int` to `Integer` can we also update the `ClassificationRequestBuilder` accordingly?

Reply:
done